<launch>
	
	<node name="main" pkg="kinect_controller" type="main" required="true"/>

	<node name="drone_controller" pkg="kinect_controller" type="drone_controller" required="true"/>

  	<node pkg="openni_tracker" name="openni_tracker" type="openni_tracker" output="screen">
	    	<param name="fixed_frame" value="openni_depth_frame" />
  	</node>
	
	<node pkg="image_view" name="image_view" type="image_view" args="image:=/camera/rgb/image_color">
	</node>
	<node name="ardrone_driver" pkg="ardrone_autonomy" type="ardrone_driver" output="screen" clear_params="true">
	    <param name="outdoor" value="0" /> <!-- If we are flying outdoors, will select more aggressive default settings -->
	    <param name="flight_without_shell" value="0" /> <!-- Changes internal controller gains if we are flying without the propeller guard -->
	    
	    <param name="altitude_max" value="3000" /> <!-- in millimeters = 3 meters = 9' -->
        <param name="altitude_min" value="50" /> <!-- in millimeters = 5cm = 2" -->
        <param name="euler_angle_max" value="0.1" /> <!-- maximum allowable body angle in radians = 5 degrees -->
        <param name="control_vz_max" value="200" /> <!-- maximum z velocity in mm/sec = 0.2m/sec -->
	    <param name="control_yaw" value="0.7" /> <!-- maximum rotation rate in radians/sec = 40 degrees per second (1/9 rev/sec) -->
	</node>
</launch>
